mappings/net/minecraft/world/level/storage/MapStorageBase.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS hm
+CLASS hm net/minecraft/world/level/storage/MapStorageBase
```

Comment:
Weird name but it's litearlly only used as a superclass of MapStorage and I think it's still like this all the way in 1.14

Reply:
Hm 🤔